analysis/processor/process.go

```diff
@@ -0,0 +1,58 @@
+package processor
+
+import (
+	"fmt"
+
+	"github.com/deepsourcelabs/cli/types"
+)
+
+// Processor interface to receive analysis post-processors.
+type IProcessor interface {
+	Name() string
+	Process([]string, *types.Issue, *[]types.Issue) error
+}
+
+// ConfigureProcessors struct contains the data needed to process the analysis results.
+type ConfigureProcessors struct {
```

Comment:
I guess the below workflow would be the way this can be used:

```go
	processor := processor.ConfigureProcessors{
		CodePath:   a.SourcePath,
		Processors: []processor.IProcessor{skip_cq, source_code_load},
	}
        processedResult :=processor.ProcessAnalyzerReport()
```